src/components/SampleUnitPopups/CollectSampleUnitPopup/CollectSampleUnitPopup.js

```diff
@@ -15,16 +15,18 @@ const CollectSampleUnitPopup = ({ rowRecord, recordProfileSummary }) => {
 
   const sampleUnitsWithPopup = sortArray(collect_records).map((record, index) => {
     const { name, sample_date, observers, management_name } = record
-    const popupTitle = `${sample_unit_method} ${name}`
 
+    const popupTitle = `${sample_unit_method} ${name}`
     const transectNumberLabel = name || language.pages.usersAndTransectsTable.missingLabelNumber
+    const keyName = transectNumberLabel + index
```

Comment:
is there a unique id that can be used from `record` instead of `index`? (React doesnt like you using index for the key value)

Reply:
no unique id for them. I have some options

1) I bypass the eslint error by doing as above
2) I have to make a very long string name with different properties (its not guaranteed they are unique)
3) Use random id generations (is it legal >.<?) 

Any thoughts?

Reply:
3 is illegal
Im guessing it complains about `transectNumberLabel` not being unique.... because they are sometimes undefined (and use the same languege file text)

Reply:
In that case just do what you were doing. Its silly they wont let you silence the warning explicitly.

Reply:
okay, I change to option 2 for now, it might be very edge case to get the same key names.

Reply:
yeah it couldn't tell. thanks I will merge with option 2